lib/ansible/utils/display.py

```diff
@@ -430,7 +430,7 @@ def banner(self, msg, color=None, cows=True):
         try:
             star_len = self.columns - get_text_width(msg)
         except EnvironmentError:
-            star_len = self.columns - len(msg)
+            star_len = min(self.columns, 79) - len(msg)
```

Comment:
self.columns should already reflect this calculation instead of having to do it for each instance of it's use

Reply:
fair enough

Reply:
im thinking the best way to handle this is to override textwrap.wrap into a noop when no tty is present, playing with column width leads to many corner cases.

Reply:
```
    def wrap(msg, size=70, **kwargs):
        if os.istty(1):
            wrapped = textwrap.wrap(msg, size, kwargs)
        else:
            wrapped = msg
        return msg
```
and change existing textwrap.wrap calls for self.wrap